doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1860,6 +1860,8 @@ Tactic notations allow customizing the syntax of tactics.
         we want to support all these?
         @JasonGross's opinion here: https://github.com/coq/coq/pull/11718#discussion_r415387421
 
+     .. todo "term" is now available, not sure it's worth documenting
```

Comment:
Please don't leave a comment like this without also mentioning the issue with the `term` entry that we discussed (and why users should favor `constr` in most cases).